rdp-proto/src/ntlm/messages/server/authenticate.rs

```diff
@@ -150,15 +138,12 @@ where
     Ok(mic)
 }
 
-fn get_authenticate_message_fields(
+fn process_message_fiels(
```

Comment:
typo: rename to `process_message_fields`

Reply:
Thanks. Done